### PR TITLE
Add link to Waffle board in README

### PR DESCRIPTION
**Pull request by jessieay**
* Also add info for people with questions
* Fixes 1/3 of #49

### CODE REVIEW

**Comment by jessieay** [comment]
I will be interested to see if this auto-closes #49, which is not what we want since that issue contains a list

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,15 +7,31 @@ To that end, we follow the [18F Code of
...
+
 ### Git Protocol
 
-To contribute to this project, people internal to 18F can create a branch and submit a pull request. If you are external to 18F, you can fork the repository and submit a pull request that way.
+To contribute to this project, people internal to 18F can create a branch and
+submit a pull request. If you are external to 18F, you can fork the repository
+and submit a pull request that way.
 
-We are minimizing commits on the `master` branch by squashing commits and force pushing them to `master`. 
+We are minimizing commits on the `master` branch by rebasing and squashing
```
Comment (ccostino) [inline CONTRIBUTING.md]: Should we talk about doing a merge --squash of any given branch into master instead?  It's a simpler workflow and takes care of squashing all of your changes from the branch and staging them for a single new commit (but doesn't automatically commit them, you still do that yourself) into whatever branch you're merging into, e.g., master.

Comment (jessieay) [inline CONTRIBUTING.md]: ooh! I remember you talking about that. Maybe you should open a separate PR with step-by-step instructions on the flow you use so people can follow it? Here we are linking to the thoughtbot steps, which is a rebase/squash, but am happy to change the flow as long as we write out detailed instructions!

Comment (ccostino) [inline CONTRIBUTING.md]: Can do!  Let's chat about it in person first to make sure it makes sense and I can walk folks through it, then get some formal instructions written up afterward. :-)

**Comment by gemfarmer** [comment]
Is the waffle board public?

File: README.md [hunk 2]
```diff
@@ -11,6 +11,24 @@ Mrs. Landingham will 18F employees about working in the federal government, how
 
 Please read the [contribution guidelines](CONTRIBUTING.md) before submitting a pull request.
 
+### Not sure what to work on?
+
+Check out the [Dolores Bot Waffle.io
+board](https://waffle.io/18F/dolores-landingham-bot) and pick out something in
+the "Up next" column. Make sure you move the feature you're working on the "In
+progress" so we all know who is working on what.
```
Comment (gemfarmer) [inline README.md]: `Make sure you move the feature you're working on the "In
progress" and assign it to yourself so we all know who is working on what.`

**Comment by jessieay** [comment]
@gemfarmer I think it's public? Is that a problem?

File: README.md (hunk 2 above)
Comment (jessieay) [inline README.md]: :+1:

**Comment by jessieay** [comment]
@gemfarmer updated

**Comment by gemfarmer** [comment]
:ship:

**Pull request merged**